app/actors/hyrax/actors/file_set_actor.rb

```diff
@@ -21,7 +22,8 @@ def create_content(file, relation = :original_file, from_url: false)
         # If the file set doesn't have a title or label assigned, set a default.
         file_set.label ||= label_for(file)
         file_set.title = [file_set.label] if file_set.title.blank?
-        return false unless file_set.save # Need to save to get an id
+        @file_set = perform_save(file_set) # Need to save to get an id # TODO: change this to `save(file_set)` when fully valkyrized
```

Comment:
what's holding this back at this point? it might be helpful to replace "fully valkyrized" with a description of the gap.

Reply:
#perform_save does the switching between AF and Valkyrie approaches.  #save is pure valkyrie.  So when we are at a point where we are passing valkyrie resources and no longer need to switch between the two methods, the changes would be...

* change the method call here from `perform_save(file_set)` to `save(file_set)`.  For `perform_save`, file_set can be either an AF object or a resource.  For `save`, file_set must be a resource.
* remove any private methods marked `TODO: Remove this method when fully valkyrized`